estimagic/dashboard/monitoring_app.py

```diff
@@ -136,10 +157,10 @@ def _create_cds_for_monitoring_app(start_params):
     crit_data = {"iteration": [], "criterion": []}
     criterion_history = ColumnDataSource(crit_data, name="criterion_history_cds")
 
-    param_names = start_params["name"].tolist()
+    param_ids = [id_ for id_list in group_to_param_ids.values() for id_ in id_list]
     params_data = {"iteration": []}
-    for name in param_names:
-        params_data[name] = []
+    for id_ in param_ids:
```

Comment:
Here a dictionary comprehension would be more readable

Reply:
Done (0081c97).